docs/source/index.md

```diff
@@ -0,0 +1,52 @@
+
+Because it doesn't assume anything beyond the official GraphQL specification, Apollo works with every GraphQL server implementation, for every language. It doesn't impose any requirements on your schema either; if you can send a query to a standard GraphQL server, Apollo can handle it. You can find a list of GraphQL server implementations on [graphql.org](http://graphql.org/code/#server-libraries).
+
+<h2 id="goals">Project goals</h2>
+
+Apollo Client is a JavaScript client for GraphQL. We built Apollo Client to be:
+
+1. **Incrementally adoptable**, so that you can drop it into an existing JavaScript app and start using GraphQL for just part of your UI.
+2. **Universally compatible**, so that Apollo works with any build setup, any GraphQL server, and any GraphQL schema.
+2. **Simple to get started with**, you can just read a small tutorial and get going.
+3. **Inspectable and understandable**, so that you can have great developer tools to understand exactly what is happening in your app.
+4. **Built for interactive apps**, so your users can make changes and see them reflected in the UI immediately.
+5. **Community driven**, so that you can be confident that the project will grow with your needs. Apollo packages are co-developed with production users from the start, and all projects are planned and developed in the open on GitHub so that there aren't any surprises.
+
+Apollo Client does more than simply run your queries against your GraphQL server. It analyzes your queries and their results to construct a client-side cache of your data, which is kept up to date as further queries and mutations are run. This means that your UI can be internally consistent and fully up-to-date with the state on the server with the minimum number of queries required.
+
+<h2 id="comparison">Other GraphQL clients</h2>
+
+If you are deciding whether to use `apollo-link` or some other GraphQL client, it's worth considering the [goals](#goals) of the project, and how they compare. Here are some additional points:
+
+ - [Relay](https://facebook.github.io/relay/) is a performance-oriented and highly opinionated GraphQL client built by Facebook for their mobile applications. It focuses on enabling the co-location of queries and components, and is opinionated about the design of your GraphQL schema, especially in the case of pagination. Apollo has an analogous set of features to Relay, but is designed to be a general-purpose tool that can be used with any schema or any frontend architecture. Relay's coupling to a specific architecture enables some benefits but with the loss of some flexibility, which also lets the Apollo community iterate more rapidly and quickly test experimental features.
+ - [Lokka](https://github.com/kadirahq/lokka) is a simple GraphQL Javascript client with a basic query cache. Apollo is more complex, but includes a much more sophisticated cache and set of advanced features around updating and refetching data.
```

Comment:
How about referencing apollo-fetch instead of Lokka? Both function as thin clients and apollo-fetch is currently maintained

Reply:
I think should just remove this section IMO, doesn't make sense to reference Relay here.